src/lib/web-component/log-view/log-view-element.js

```diff
@@ -52,7 +52,7 @@ class LogView extends HTMLElement {
   }
 
   openDetailSidebar(logDetails) {
-    const logString = JSON.stringify(logDetails);
+    const logString = JSON.stringify(logDetails, null, 4);
```

Comment:
Nit, 2 (instead of 4) may be enough for the indentation, and it would spare a bit of horizontal space.